lms/services/aes.py

```diff
@@ -38,6 +38,9 @@ def build_iv():
         # Using the cryptographic secure Cryptodome.Random instead of the stlib random package.
         return Random.new().read(AES.block_size)
 
+    class AESSecret:
```

Comment:
I'm using this in the settings definition. 

Arguably this could be just "secret" over there but this is cheap as well.